css/main.css

```diff
@@ -45,24 +42,17 @@
 }
 
 input[type="file"] {
-  margin: 10px;
+  margin: 10px 0;
 }
+
 fieldset {
   margin: 10px 0px;
 }
+
 #output-display {
-  height: 100%;
-  text-align: center;
-  padding: 15px auto;
-}
-#output-display canvas {
-  width: 70%;
```

Comment:
I suspect this width needs to added again to stop the output image from changing size when the ratio is changed.

Reply:
Yes my mistake, I suppose that resize was the functionality, I'll make a new commit.